cns/restserver/util.go

```diff
@@ -853,7 +853,9 @@ func (service *HTTPRestService) isNCWaitingForUpdate(
 			"Skipping GetNCVersionStatus check from NMAgent", ncVersion, ncid)
 		return true, types.NetworkContainerVfpProgramPending, ""
 	}
-	nmaProgrammedNCVersionStr, ok := ncVersionList[ncid]
+	// accept both upper and lower GUID from ncid(Swift_GUID)
+	// check each ncid in lower case if it's in ncVersionList
```

Comment:
let's also add a code to store the keys in nmaNCs as lower case NCIDs ( in the function that calls isNCWaitingForUpdate )

Also need to add UTs that validates ths